ART_NOTES.md

```diff
@@ -0,0 +1,112 @@
+
+# **Art Notes**
+
```

Comment:
this doc should be linked to by the readme. People don't know we have other docs beside the readme

Reply:
Speaking of docs, this md should reside in the Documentation folder